src/tests/ftest/io/ior_small.py

```diff
@@ -48,4 +48,30 @@ def test_ior_small(self):
 
         :avocado: tags=all,daosio,small,pr,hw,iorsmall
         """
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+
+        # run tests for different variants
+        self.ior_cmd.flags.update(flags[0])
+        for oclass in obj_class:
+            self.ior_cmd.daos_oclass.update(oclass)
+            for api in apis:
+                self.ior_cmd.api.update(api)
+                for test in transfer_block_size:
+                    # update transfer and block size
+                    self.ior_cmd.transfer_size.update(test[0])
+                    self.ior_cmd.block_size.update(test[1])
+                    # run ior
+                    self.run_ior_with_pool()
```

Comment:
In the case of the `256B` transfer size, it doesn't look like the user is actually notified that the case is skipped.  This function now returns `"Skipping the case for transfer_size=256B"`, but it is not displayed.  One possible solution would be to have the `run_ior_with_pool` method log/print the message and just return `None`.

Reply:
Well is it okay if we take care of that in a different pr. Landing of this one can save decent amount of time. If you are fine we can go ahead with it. I will add the one liner that in my another PR which is not yet merged.